### PR TITLE
122 Publish All Projects

### PR DESCRIPTION
**Pull request by lombad**
It should be possible to publish all projects an all the links with one single command e.g. gittt publish -all

AC:
- [x] All data gets published with a single command

### CODE REVIEW

File: app.ts [hunk 1]
```diff
@@ -238,26 +238,46 @@ export class App {
...
+        return {
+          name: project.name,
+          meta: project.meta,
+          records: project.records,
+          integrationLinks: links
+        };
+      })
+    );
+
+    if (projectIntegrationLinks.length === 1 && projectIntegrationLinks[0].integrationLinks.length === 0) {
```
Comment (matthiaseigner) [inline app.ts:279]: what If the second project in the array does not have any links? I will not crash but no error would be displayed - correct?

File: test/unit/app.test.ts [hunk 2]
```diff
@@ -4828,7 +4828,109 @@ describe("App", function () {
...
           exitStub.restore();
         });
+      });
+    });
+
+    describe("Publish All Projects", function () {
+      it("should publish all projects to Multipie endpoint [non interactive]", async function () {
+        const mockedHelper: any = Object.assign({}, emptyHelper);
+
+        // const getProjectByNameStub = sinon.stub().returns({
```
Comment (matthiaseigner) [inline test/unit/app.test.ts:4838]: could this be removed?

**Review by matthiaseigner — CHANGES_REQUESTED** [review]
2 small questions

File: app.ts (hunk 1 above)
Comment (lombad) [reply app.ts:279]: Cool finding, I added a warning in 804d38e4c40e13b6a911b448b9828d7dfcd49b01

File: test/unit/app.test.ts (hunk 2 above)
Comment (lombad) [reply test/unit/app.test.ts]: removed in 2d2539f6cf36ecb1b0466673806f0ed0235b3f25

**Review by matthiaseigner — APPROVED** [review]

**Pull request merged**